### ISSUE TITLE
Create a linux client installation guide

### ISSUE THREAD

**Issue by Vysp3r** [tiltedphoques/TiltedEvolution#366]
## Description

It would be really nice to have a guide on how to install this mod on linux to help linux player out. It would be great for people using the Steam Deck too.

**Comment by extract** [comment]
Agreed, would be nice with a nicer install, like a hook or something, as currently I just put the skyrimtogether mod in Skyrim directory and then renaming SkyrimTogether.exe to SkyrimSELauncher.exe...

**Comment by Miuzorosaurus** [comment]
I've managed to make it to work with mods. If anyone is looking how to do this on Linux:

Use steamtinkerlaunch as compatibility tool (download it, it doesn't come with Steam), then launch Skyrim SE through it, and set up Vortex just like in the guide (just don't log in and install your mods by drag and drop to the mod launcher, I had issues with installing mods through nexusmods directly). 

Then follow the guide up until the point where you have to create a tool through Vortex to launch SkyrimTogether.exe - if you do that, you're likely going to get an error saying how you need windows 8.1 and above to run the mod. Instead, you gotta close Vortex, go to "game menu" in steamtinkerlaunch skyrim SE and set the custom command to run `Data/Skyrim Together/SkyrimTogether.exe`

Then continue following the guide.

One issue I had when modding on linux through steamtinkerlaunch in general was that I couldn't load any .esp mods - that's because, at least on my end, vortex created its own compatibility data folder, which was located in `~/.config/steamtinkerlaunch/vortex/compatdata/pfx/drive_c/users/steamuser/AppData/Local/Skyrim Special Edition` where the load order files are at - copy these over and get them to `~/.steam/steam/steamapps/compatdata/489830/pfx/drive_c/users/steamuser/AppData/Local/Skyrim Special Edition`

I know that this is a suggestion and not a call for help, but I'm just leaving this here in case someone is struggling to have the mod work on Linux. I've done this on Arch (btw), but it should work on most distros.

**Comment by Vysp3r** [comment]
> I've managed to make it to work with mods. If anyone is looking how to do this on Linux:
> 
> Use steamtinkerlaunch as compatibility tool (download it, it doesn't come with Steam), then launch Skyrim SE through it, and set up Vortex just like in the guide (just don't log in and install your mods by drag and drop to the mod launcher, I had issues with installing mods through nexusmods directly).
> 
> Then follow the guide up until the point where you have to create a tool through Vortex to launch SkyrimTogether.exe - if you do that, you're likely going to get an error saying how you need windows 8.1 and above to run the mod. Instead, you gotta close Vortex, go to "game menu" in steamtinkerlaunch skyrim SE and set the custom command to run `Data/Skyrim Together/SkyrimTogether.exe`
> 
> Then continue following the guide.
> 
> One issue I had when modding on linux through steamtinkerlaunch in general was that I couldn't load any .esp mods - that's because, at least on my end, vortex created its own compatibility data folder, which was located in `~/.config/steamtinkerlaunch/vortex/compatdata/pfx/drive_c/users/steamuser/AppData/Local/Skyrim Special Edition` where the load order files are at - copy these over and get them to `~/.steam/steam/steamapps/compatdata/489830/pfx/drive_c/users/steamuser/AppData/Local/Skyrim Special Edition`
> 
> I know that this is a suggestion and not a call for help, but I'm just leaving this here in case someone is struggling to have the mod work on Linux. I've done this on Arch (btw), but it should work on most distros.

The thing is, it's just not working for me using the way you just said. I want to do it that way because it sounds like the best way, but I after 4 hours of trying I gave up. I just dropped the mod manually in the folder and it now works. At least it works for me, but it's kinda annoying that STL is not working for me.

**Comment by lyn1337** [comment]
It would be useful if those, who got it working on their Linux based system without mod managers, would post their dir tree.
